src/components/FAQ-page/Header.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
Delete this file! We do not need different headers for each page. The gist of React is to create reusable components :)

Reply:
Should I include Navbar component instead of this ?

Reply:
Already added: https://github.com/WooTechnology/frontend/pull/12#discussion_r254815899